### ISSUE TITLE
ipsec checknss, custom directory  not recognized

### ISSUE THREAD

**Issue by berkrie** [libreswan/libreswan#44]
I set up Openstack with VPNaaS.

During creation process of the VPN this command is executed:
ip netns exec qrouter-664940e8-6139-4c36-8fcc-ee9e06bd5212 ipsec checknss /var/lib/neutron/ipsec/664940e8-6139-4c36-8fcc-ee9e06bd5212/etc

The db files are always added to the  directory /etc/ipsec.d
Looking at the bash script "ipsec"  i saw that the variable IPSEC_NSSDIR_SQL is set only at the beginning of the script.
If a custom directory is specified, the variable  IPSEC_NSSDIR_SQL is not changed.

Adding IPSEC_NSSDIR_SQL="sql:${IPSEC_NSSDIR}" resolved the issue.

```initnss|--initnss|checknss|--checknss)
        if [ -n "${2}" ]; then
            # A lot of nss commands use -d or --configdir to specify
            # NSS db location
            if [ "${2}" = "-d" -o "${2}" = "--configdir" ]; then
                IPSEC_NSSDIR="${3}"
            else
                IPSEC_NSSDIR="${2}"
            fi
       +IPSEC_NSSDIR_SQL="sql:${IPSEC_NSSDIR}"
        fi
        if [ ! -d "${IPSEC_NSSDIR}" ]; then
            mkdir -p "${IPSEC_NSSDIR}"
        fi
        # if we have old database
        if [ -f "${IPSEC_NSSDIR}/cert8.db" -o \
````

**Comment by bleve** [comment]
Thank you, fixed for 3.16.

**Comment by letoams** [comment]
will be in 3.15
